Wrapping/Python/Examples/use_case_02.py

```diff
@@ -36,6 +45,35 @@ def import_csv_xyz_vertices_2(file_path: str):
     dca.addOrReplaceDataContainer(dc)
     # Write the Vertex Geometry out to a .dream3d file.
     err = simpl_helpers.WriteDREAM3DFile('vert_geom_2.dream3d', dca, True)
+    
+    # Create Cell AttributeMatrix
+    tupleDims = simpl.VectorSizeT([1, 37989])
```

Comment:
Where did the '37989' value come from? this value should match the number of vertices read from the input file.

Maybe something like
```
    numVerts = len(verts_np)
    # Create Cell AttributeMatrix
    tupleDims = simpl.VectorSizeT([1, numVerts])
```